ipaclient/plugins/dns.py

```diff
@@ -431,11 +489,12 @@ def output_for_cli(self, textui, output, *args, **options):
         }
 
         result = output.get('result', {})
-        for key in ('ipa_records', 'location_records'):
-            if result.get(key):
-                textui.print_indented(u'{}:'.format(labels[key]), indent=1)
-                for val in sorted(result[key]):
-                    textui.print_indented(val, indent=2)
-                textui.print_line(u'')
+
+        self._standard_output(textui, result, labels)
+
+        out = options.get('out')  # output to file
+        if out:
+            with open(out, "w") as f:
```

Comment:
I think it'd be nice if we had try-catch block here to handle potential errors with file operations.